### PR TITLE
docs: update README for new binary name

### PR DESCRIPTION
**Pull request by efekarakus**
_By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice._

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,80 +1,74 @@
...
+Want to learn more about what's happening? Check out our documentation [https://aws.github.io/copilot-cli/](https://aws.github.io/copilot-cli/) for a getting started guide, learning about Copilot concepts, and a breakdown of our commands. 
 
 ## We need your feedback 🙏
 
-The ECS CLI v2 is in developer preview, meaning we want to know what works, what doesn't work, and what you want! Have any feedback at all? Drop us an [issue](https://github.com/aws/amazon-ecs-cli-v2/issues/new) or join us on [gitter](https://gitter.im/aws/amazon-ecs-cli-v2?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge).
+The AWS Copilot CLI is in beta, meaning that you can expect our command names to be stable as well as the shape of our 
+infrastructure patterns. However, we want to know what works, what doesn't work, and what you want! 
+Have any feedback at all? Drop us an [issue](https://github.com/aws/copilot-cli/issues/new) or join us on [gitter](https://gitter.im/aws/amazon-ecs-cli-v2?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge).
```
Comment (kohidave) [inline README.md:71]:
```suggestion
Have any feedback at all? Drop us an [issue](https://github.com/aws/copilot-cli/issues/new) or join us on [gitter](https://gitter.im/aws/copilot-cli?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge).
```

Comment (efekarakus) [reply README.md:71]: Updated also the badge at the top of the README!

File: README.md [hunk 2]
```diff
@@ -1,80 +1,74 @@
...
-  --app-type 'Load Balanced Web App' \
-  --dockerfile './Dockerfile'        \
-  --port 80                          \
+$ git clone git@github.com:aws-samples/aws-copilot-sample-service.git demo-app
+$ cd demo-app
+$ copilot init --app demo                \
+  --svc api                              \
+  --svc-type 'Load Balanced Web Service' \
+  --dockerfile './Dockerfile'            \
+  --port 80                              \
```
Comment (kohidave) [inline README.md:53]: We can get rid of the port, as well. 
```suggestion
```

Comment (efekarakus) [reply README.md]: Done!

**Pull request merged**